src/main/java/frc/robot/LogitechGamingPad.java

```diff
@@ -26,230 +26,207 @@
-	 * 
-	 * @return The left Y value.
-	 */
-
-	public double getLeftAnalogY()
-	{
-		return getLeftAnalogY();
-	}
-
 	/**
 	 * Read the state of the left analog button.
 	 * This is the button part of the analog.
 	 * 
 	 * @return The state of the button.
 	 */
 
-	public boolean getLeftAnalogButton()
-	{
-		return gamepad.getRawButton(9);
-	}
+  public boolean getLeftBumper() {
+    return gamepad.getRawButton(LEFT_BUMPER);
+  }
 
 	/**
-	 * Read the state of the left bumper.
-	 * The bumpers are the two upper buttons of the four on the
-	 * top of the gamepad. They are either pressed or released.
+	 * Get the value of the left trigger. The trigger returns a double [0, 1]
+	 * depending on how much the trigger is pressed. 0 is nothing pressed
+	 * and 1 is the trigger pressed all the way. 
+	 * 
+	 * This method can be used if something wants to be varied depending on
+	 * how much the trigger is pressed.
 	 * 
-	 * @return The state of the bumper.
+	 * @return The value of the left trigger.
 	 */
 
-	public boolean getLeftBumper()
-	{
-		return getLeftBumper();
-	}
+  public double getLeftTriggerValue() {
+    return gamepad.getRawAxis(LEFT_TRIGGER_BUTTON);
+  }
 
+	
 	/**
-	 * Read the state of the left trigger.
-	 * The triggers are the two lower buttons of the four on
-	 * the top of the gamepad. How much they are pressed can vary.
+	 * Get the value of the left joystick along the x axis. The trigger returns a double [-1, 1]
+	 * depending on how far the joystick is moved. 0 is nothing when untouched 
+	 * and 1/-1 is the joystick moved all the way left or right respectively. 
 	 * 
-	 * @return The state of the trigger.
+	 * This method can be used for precise control over a subsystem.
+	 * 
+	 * @return The value of the left joystick along the x axis. 
 	 */
 
-	public boolean getLeftTrigger()
-	{
-		return getLeftTrigger();
-	}
+  public double getLeftAnalogXAxis() {
+    return gamepad.getRawAxis(LEFT_ANALOG_X_AXIS);
+  }
 
 	/**
-	 * Get the value of the left trigger. The trigger returns a double [0, 1]
-	 * depending on how much the trigger is pressed. 0 is nothing pressed
-	 * and 1 is the trigger pressed all the way. 
+	 * Get the value of the left joystick along the y axis. The trigger returns a double [-1, 1]
+	 * depending on how far the joystick is moved. 0 is nothing when untouched 
+	 * and 1/-1 is the joystick moved all the way up or down respectively. 
 	 * 
-	 * This method can be used if something wants to be varied depending on
-	 * how much the trigger is pressed.
+	 * This method can be used for precise control over a subsystem.
 	 * 
-	 * @return The value of the left trigger.
+	 * @return The value of the left joystick along the y axis. 
 	 */
 
-	public double getLeftTriggerValue()
-	{
-		return gamepad.getRawAxis(2);
-	}
+  public double getLeftAnalogYAxis() {
+    return gamepad.getRawAxis(LEFT_ANALOG_Y_AXIS);
+  }
 
 	/**
 	 * Get the X value of the right analog.
 	 * 
 	 * @return The right X value.
 	 */
```

Comment:
as its corresponding method has been removed, please remove this comment block as well